pythonfmu/pythonfmu-export/cpp/PySlaveInstance.cpp

```diff
@@ -302,6 +302,8 @@ void PySlaveInstance::GetBoolean(const cppfmu::FMIValueReference* vr, std::size_
 
 void PySlaveInstance::GetString(const cppfmu::FMIValueReference* vr, std::size_t nvr, cppfmu::FMIString* values) const
 {
+    clearStrBuffer();
```

Comment:
I think this should be inside the lambda function (with the GIL captured) as the Python function `Py_DECREF` is called when clearing the buffer.

Reply:
Indeed. I was think while I did  that that it did not call any Python code..